### ISSUE TITLE
java.lang.NoClassDefFoundError trying to use MultiMatchQueryBuilder$Type

### ISSUE THREAD

**Issue by alex1712** [sksamuel/elastic4s#965]
When I try to use a `multiMatchQuery` setting the `matchType` to any value I get following exception:
```
play.api.http.HttpErrorHandlerExceptions$$anon$1: Execution exception[[RuntimeException: java.lang.NoClassDefFoundError: Could not initialize class org.elasticsearch.index.query.MultiMatchQueryBuilder$Type]]
	at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:293)
	at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:220)
	at play.api.GlobalSettings$class.onError(GlobalSettings.scala:160)
	at play.api.DefaultGlobal$.onError(GlobalSettings.scala:188)
	at play.api.http.GlobalSettingsHttpErrorHandler.onServerError(HttpErrorHandler.scala:100)
	at play.core.server.netty.PlayRequestHandler$$anonfun$2$$anonfun$apply$1.applyOrElse(PlayRequestHandler.scala:100)
	at play.core.server.netty.PlayRequestHandler$$anonfun$2$$anonfun$apply$1.applyOrElse(PlayRequestHandler.scala:99)
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:346)
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:345)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:36)
Caused by: java.lang.RuntimeException: java.lang.NoClassDefFoundError: Could not initialize class org.elasticsearch.index.query.MultiMatchQueryBuilder$Type
...
```

This is the code to generate my query:
```
 multiMatchQuery(q.text).operator("AND").matchType("cross_fields")
   .fields(Map(
     "field_1" -> 3f,
     "field_2" -> 2f,
   ))
```
I also tried this `.matchType(MultiMatchQueryBuilder.Type.CROSS_FIELDS)` with the same result.

Does anyone have the same problem? Do you know a workaround for this? Or am I doing something wrong? :)

Thank you very much.

**Comment by sksamuel** [comment]
Which version of e4s?

**Comment by alex1712** [comment]
latest from 5.4.x

**Comment by alex1712** [comment]
I think is related with the fact that the Type enum is inside a java class... but don't know how to get it to work

**Comment by sksamuel** [comment]
This is fixed in the 6.0.0 http client.
